crates/uv-interpreter/src/interpreter.rs

```diff
@@ -399,6 +411,33 @@ impl Interpreter {
         &self.sysconfig_paths.stdlib
     }
 
+    /// Return the name of the Python directory used to build the path to the
+    /// `site-packages` directory.
+    ///
+    /// If one could not be determined, then `python` is returned.
+    pub fn site_packages_python(&self) -> &str {
+        if self.implementation_name() == "pypy" {
+            "pypy"
+        } else {
+            "python"
+        }
+
+        // This implementation sniffs out what the directory name
+        // might be from sysconfig, but at the time of writing, this
+        // seems a little more risky than the simpler but less robust
+        // implementation above.
+        /*
+        const FALLBACK: &str = "python";
+        let Some(base) = self.include().file_name().and_then(|name| name.to_str()) else {
+            return FALLBACK;
+        };
+        base.char_indices()
+            .take_while(|(_, ch)| ch.is_ascii_alphabetic())
+            .last()
+            .map_or(FALLBACK, |(end, ch)| &base[..end + ch.len_utf8()])
+        */
```

Comment:
I'd probably cut this part, it can exist in Git if we need it later!

Reply:
I deleted it in a separate commit so that it's actually preserved in history (and did a rebase & merge). Otherwise this would have only been preserved on GitHub maybe?